modules/exploits/linux/http/nagios_xi.rb

```diff
@@ -0,0 +1,363 @@
+class MetasploitModule < Msf::Exploit::Remote
```

Comment:
Module should include the standard comment header, so users know where the module is from and how to use it.

```ruby
##
# This module requires Metasploit: https://metasploit.com/download
# Current source: https://github.com/rapid7/metasploit-framework
##
```